app/src/main/java/com/codesoom/assignment/http/HttpMethod.java

```diff
@@ -0,0 +1,20 @@
+package com.codesoom.assignment.http;
+
+public enum HttpMethod {
+    GET("GET"),
+    POST("POST"),
+    PUT("PUT"),
+    PATCH("PATCH"),
+    DELETE("DELETE");
+
+    private String httpsStatus;
```

Comment:
`final`로 불변성을 강조해 주면 좋겠습니다.
```suggestion
    private final String httpsStatus;
```